EventListener/ExceptionSubscriber.php

```diff
@@ -92,23 +94,56 @@ public function onKernelException(GetResponseForExceptionEvent $event)
 
         $exception = $event->getException();
 
-        $this->logger->warning(
-            'Exception has been thrown.',
-            [
-                'exception_code'    => $exception->getCode(),
-                'exception_message' => $exception->getMessage(),
-                'exception_file'    => $exception->getFile(),
-                'exception_trace'   => $exception->getTraceAsString(),
-                'exception_line'    => $exception->getLine()
-            ]
+        $this->logException(
+            $exception,
+            sprintf(
```

Comment:
All extra information should go into context not into message

Reply:
All information about exception has been removed form the message and moved into context